apps/academy-wep/components/CommunityStats.tsx

```
@@ -0,0 +1,50 @@
+interface CommunityStatsProps {
+	memberCount: string | number
+}
+
+export function CommunityStats({ memberCount }: CommunityStatsProps) {
+	return (
+		<div className="relative flex justify-center md:justify-end">
+			<div className="absolute inset-0 bg-gradient-to-br from-[#7CC635]/20 to-blue-500/20 rounded-full blur-xl transform rotate-6 scale-95"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)